docker.d/worker.yml

```diff
@@ -0,0 +1,164 @@
+work_dir: { "$eval": "WORK_DIR" }
+artifact_dir: { "$eval": "ARTIFACTS_DIR" }
+aiohttp_max_connections: 10
+checksums_digests:
+  - 'sha512'
+  - 'sha256'
+blobs_needing_prettynaming_contents:
+  - 'target.test_packages.json'
+taskcluster_scope_prefix: { "$eval": "TASKCLUSTER_SCOPE_PREFIX" }
+verbose: { "$eval": "VERBOSE == 'true'" }
+zip_max_file_size_in_mb: 300
+bucket_config:
+  $match:
+    'COT_PRODUCT == "firefox" && (ENV == "dev" || ENV == "stage")':
+      dep:
```

Comment:
@rail should we rename `dep` to `fake-prod` and `dep-partners` to `fake-prod-partners`?